recipes/shield/all/conanfile.py

```diff
@@ -0,0 +1,32 @@
+from conans import ConanFile, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+class ShieldConan(ConanFile):
+    name = "shield"
+    topics = ("utility", "warnings", "suppression")
+    description = "C++ warning suppression headers."
+    settings = "os", "compiler", "build_type", "arch"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/holoplot/shield"
+    license = "MIT"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+
+    def package(self):
+        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
+        self.copy(pattern="*", dst="include", src=os.path.join(self._source_subfolder, "include"))
+
+    def package_id(self):
+        self.info.header_only()
+
+    def package_info(self):
+        self.cpp_info.names["cmake_find_package"] = "shield"
+        self.cpp_info.names["cmake_find_package_multi"] = "shield"
```

Comment:
There is no CMake config file upstream, so these values shouldn't be enforced.

Reply:
So, removing them would still work, right? I can still make a PR. I think I copied this from one of the recommended recipes to take as an example.

Reply:
yes